GUI_L02_2/Square.java

```diff
@@ -0,0 +1,20 @@
+package GUI_L02_2;
+
+class Square extends Figure {
+    private double side;
+
+    public Square(double side, String color) {
```

Comment:
Optional: if Square extends Rectangle, then you will not have to implement the getArea and getPerimeter function